### ISSUE TITLE
Library not working in PageView.builder

### ISSUE THREAD

**Issue by billizzard2** [karvulf/flutter-reorderable-grid-view#113]
Example from description work well. But if i put it in PageView.builder, it's not working. I can drag item, but can't reorder

```
return PageView.builder(
    controller: controller,
    onPageChanged: (index) {
      changePage(index);
    },
    itemBuilder: ((context, index) {
      return ReordableExample(); // Widget from documentation

    })
  );
```

**Comment by karvulf** [comment]
Hello @billizzard2 ,
thank you for opening the issue.
I have to check this, sounds like a bug

**Comment by karvulf** [comment]
Probably this issue is fixed now and will work with version `5.3.1`. See more on the other issue #121
